scripts/devnet/lotus.dockerfile

```diff
@@ -13,6 +13,7 @@ RUN git clone --depth 1 --branch FIP0079-testing https://github.com/filecoin-pro
 
 RUN CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__" \
     CGO_CFLAGS="-D__BLST_PORTABLE__" \
+    FFI_USE_BLST_PORTABLE="1" \
```

Comment:
Should we have a mergeable PR with this?

Reply:
Will do

Reply:
https://github.com/ChainSafe/forest/pull/4327